src/core/GameKeyHandler.ttslua

```diff
@@ -98,14 +99,18 @@ function takeClueFromLocation(playerColor, hoveredObject)
     return
   end
 
+  local clickableClues = optionPanelApi.getOptions()["useClueClickers"]
   local playerName = Player[playerColor].steam_name
-  local matColor = playmatApi.getMatColor(playerColor) 
-  local pos = playmatApi.transformLocalPosition({x = -1.12, y = 0.05, z = 0.7}, matColor)
-  local rot = playmatApi.returnRotation(matColor)
-
-  if cardName == "" or cardName == nil then
-    cardName = "nameless card"
+  local matColor = playmatApi.getMatColor(playerColor)
+  local pos = nil
+  if clickableClues then
+    pos = {x = 0.49, y = 2.66, z = 0.00}
+    playmatApi.updateClueClicker(playerColor, playmatApi.getClueCount(clickableClues, playerColor) + 1)
+  else
+    pos = playmatApi.transformLocalPosition({x = -1.12, y = 0.05, z = 0.7}, matColor)
   end
+  
+  local rot = playmatApi.returnRotation(matColor)
```

Comment:
I think you need to add a replacement effect to the physical move of the clue token - see LOC 116-121 in your code (like moving it to a trash can instead).